flake.nix

```diff
@@ -582,6 +584,35 @@
             '';
           };
         };
+      plutarchWebsite = system:
+        let
+          pkgs = nixpkgsFor system;
+          configFile = (pkgs.formats.yaml { }).generate "emanote-configFile" {
+            template.baseUrl = "/"; # Use this when pushing to github.io: "/plutarch/";
+          };
+          configDir = pkgs.runCommand "emanote-configDir" { } ''
+            mkdir -p $out
+            cp ${configFile} $out/index.yaml
+          '';
+        in
+        pkgs.runCommand "plutarch-docs-html" { }
+          ''
+            mkdir $out
+            ${inputs.emanote.defaultPackage.${system}}/bin/emanote \
+              --layers "${self}/docs;${configDir}" \
+              gen $out
+          '';
+      plutarchWebsiteApp = system:
+        rec {
+          type = "app";
+          # '' is required for escaping ${} in nix
+          script = (nixpkgsFor system).writers.writeBash "emanoteLiveReload.sh" ''
```

Comment:
How is this different from `writeShellScriptBin`?

Reply:
It's a copy-paste from [plutonomicon](https://github.com/Plutonomicon/plutonomicon/blob/f31ce5405777558ae9c7eb83c739266ca579ee98/flake.nix#L26), but now that you mention it I realize we should instead be using [`checkedShellScript`](https://github.com/Plutonomicon/plutarch/blob/aecc2050eb63ff0041576473aa3193070fe91314/flake.nix#L533) from the flake.